### PR TITLE
Add error log imgproc

### PR DESCRIPTION
**Pull request by xresende**
requires https://github.com/lnls-sirius/dev-packages/pull/1060

### CODE REVIEW

**Review by RafaelLyra8 — COMMENTED** [review]
You could use the PyDMLogLabel widget instead of the QLabel.

File: pyqt-apps/siriushla/bl_ap_imgproc/main.py [hunk 1]
```diff
@@ -305,6 +305,9 @@ def update_bl_open_led(self):
             self.pydm_led.value_changed(status_bl)
             if old_val != status_bl:
                 self.end_processing_cmd()
+        if self.pydm_lbl != None:
```
Comment (anacso17) [inline pyqt-apps/siriushla/bl_ap_imgproc/main.py:308]: This if doesn't seem to be needed

**Comment by anacso17** [comment]
> You could use the PyDMLogLabel widget instead of the QLabel.

@RafaelLyra8 it seems the log history is not needed here, maybe a simple label is enough...

**Review by fernandohds564 — APPROVED** [review]

File: pyqt-apps/siriushla/bl_ap_imgproc/main.py (hunk 1 above)
Comment (anacso17) [reply pyqt-apps/siriushla/bl_ap_imgproc/main.py:308]: Oh, I see. It is needed because timer is created before setting up the widgets...

**Comment by xresende** [comment]
> > You could use the PyDMLogLabel widget instead of the QLabel.
> 
> @RafaelLyra8 it seems the log history is not needed here, maybe a simple label is enough...

I did not think of log history, just an instant log for when the BL opening fails.

**Review by anacso17 — APPROVED** [review]

**Pull request merged**